pkg/kubectl/explain/formatter.go

```diff
@@ -61,6 +61,8 @@ func (f *Formatter) WriteWrapped(str string, a ...interface{}) error {
 	text := fmt.Sprintf(str, a...)
 	strs := wrapString(text, f.Wrap-f.IndentLevel)
 	for _, substr := range strs {
+		// escape literal percent sign
+		substr = strings.Replace(substr, "%", "%%", -1)
```

Comment:
Mind adding a test case covering it?

Reply:
Hmm… are there other formatting directives that will cause issues? I'd prefer we stop treating this as a format string altogether than try to escape here for later layers

Reply:
Yeah, I think Jordan is right. I'd be better to just take a pre-formatted string, and call `Sprintf` from the caller, or add maybe add a `Writef` for convenience.